### PR TITLE
Fine tune maxprocs and minpeer interval

### PR DESCRIPTION
**Pull request by rlan35**
Change block rewards to 24 per block.
Add genesis string to all shards: "Harmony for One and All. Open Consensus for 10B."
Fix consensus retry logic.
Other sys/network fine tunes.

### CODE REVIEW

File: node/node_genesis.go [hunk 1]
```diff
@@ -102,6 +102,7 @@ func (node *Node) SetupGenesisBlock(db ethdb.Database, shardID uint32, myShardSt
 		GasLimit:       params.GenesisGasLimit * 1000,
 		ShardStateHash: myShardState.Hash(),
 		ShardState:     myShardState.DeepCopy(),
+		ExtraData:      []byte("Harmony for One and All, Open Consensus for 10B"),
 	}
```
Comment (alajko) [inline node/node_genesis.go]: There is a full stop not a comma, in the message. See https://github.com/harmony-one/harmony/pull/1154

**Comment by chaosma** [comment]
|
                    |
           |        |
         |-|-|      |
           |        |
           | {O}    |
           '--|     |
             .|]_   |
       _.-=.' |     |
      |    |  |]_   |
      |_.-='  |   __|__
       _.-='  |\   /|\
      |    |  |-'-'-'-'-.
      |_.-='  '========='
           `   |     |
            `. |    / \
              ||   /   \____.--=''''==--.._
              ||_.'--=='    |__  __  __  _.'
              ||  |    |    |\ ||  ||  || |                        ___
 ____         ||__|____|____| \||__||__||_/    __________________/|   |
|    |______  |===.---. .---.========''''=-._ |     |     |     / |   |
|    ||     |\| |||   | |   |      '===' ||  \|_____|_____|____/__|___|
|-.._||_____|_\___'---' '---'______....---===''======//=//////========|
|--------------\------------------/-----------------//-//////---------/
|               \                /                 // //////         /
|                \______________/                 // //////         /
|                                        _____===//=//////=========/
|==============================================================/
'----------------------------------------------------------------`

File: node/node_genesis.go (hunk 1 above)
Comment (rlan35) [reply node/node_genesis.go]: Good catch .Changed to period.

**Comment by alajko** [comment]
Genesis!

**Pull request merged**